src/spdx3/bump_from_spdx2/package.py

```diff
@@ -54,13 +53,14 @@ def bump_package(
             spdx_id,
             creation_information,
             name,
-            verified_using=integrity_methods,
-            download_location=download_location,
-            homepage=homepage,
             summary=summary,
             description=description,
             comment=comment,
+            verified_using=integrity_methods,
             originated_by=originated_by_spdx_id,
             package_purpose=package_purpose,
+            package_version=package_version,
+            download_location=download_location,
+            homepage=homepage,
```

Comment:
I'm not sure why the first black sorting (in the PR just a few minutes ago) used a different sorting than the one I did now in this PR

Reply:
I also don't know why this happens. The only thing I could imagine: By adding `package_version` black was triggered in some different way and aligned the order of arguments according to the definition of `Package.__init__()`. But I also don't know why this wasn't triggered in the first PR.